userguide/tutorials/events-to-aws-sqs.adoc

```diff
@@ -0,0 +1,117 @@
+= Configuring Killbill to Send Events to AWS SQS
+
+== Overview
+
+This document outlines the steps to configure the Killbill application to send events to an Amazon Simple Queue Service (SQS) queue.
```

Comment:
Maybe add a link to https://docs.killbill.io/latest/kill_bill_events to explain what a Kill Bill event is?